Resources/Private/Templates/ContentElements/Dynamic/Slider.html

```diff
@@ -43,30 +43,45 @@
 
               <f:if condition="{item.data.header_text} || {t3kit:isPlaceholderEnabled()}">
                 <div class="ce-slider__caption-header">
-                  <f:render partial="Header/HeaderText" arguments="{data: item.data}" />
+                  <f:render
+                    partial="Header/HeaderText"
+                    arguments="{data: item.data, table: 'tx_t3kit_slider_item', field: 'header_text'}"
+                  />
                 </div>
               </f:if>
 
               <f:if condition="{item.data.subheader_text} || {t3kit:isPlaceholderEnabled()}">
                 <div class="ce-slider__caption-subheader">
-                  <f:render partial="Header/SubHeaderText" arguments="{data: item.data}" />
+                  <f:render
+                    partial="Header/SubHeaderText"
+                    arguments="{data: item.data, table: 'tx_t3kit_slider_item', field: 'subheader_text'}"
```

Comment:
do we really need to specify `field` here, looks like `subheader_text ` is a default value?

Reply:
I thought it was wise, as it is a different table. In case the tt_content schema changes, the field names might not be the same.